### PR TITLE
Make telegram on_alert_group_action_triggered asynchronous

### PR DESCRIPTION
**Pull request by iskhakov**
# What this PR does

[send_alert_group_signal](https://github.com/grafana/oncall/blob/dev/engine/apps/alerts/tasks/send_alert_group_signal.py#L12) task is not idempotent. It launches [on_alert_group_action_triggered_async](https://github.com/grafana/oncall/blob/a2851d3f813c04768ce825c8af99e26912d80b4e/engine/apps/slack/representatives/alert_group_representative.py#L158) for slack and then might fail on [on_alert_group_action_triggered](https://github.com/grafana/oncall/blob/b2f4ffb98a940b894042df9161e209dc8a781dc1/engine/apps/telegram/alert_group_representative.py#L79) (not async) due to database DoesNotExist exception.

This PR makes telegram representative asyncronous

## Checklist

- [ ] Unit, integration, and e2e (if applicable) tests updated
- [ ] Documentation added (or `pr:no public docs` PR label added if not required)
- [ ] `CHANGELOG.md` updated (or `pr:no changelog` PR label added if not required)

### CODE REVIEW

File: engine/apps/telegram/alert_group_representative.py [hunk 1]
```diff
@@ -78,15 +80,11 @@ def on_alert_group_action_triggered(cls, **kwargs):
         from apps.alerts.models import AlertGroupLogRecord
 
         log_record = kwargs["log_record"]
```
[the remark this answers is not in the document]
Comment (iskhakov) [reply engine/apps/telegram/alert_group_representative.py:82]: just prefer split fixes and refactorings into different tasks

File: engine/apps/telegram/tasks.py [hunk 2]
```diff
@@ -215,3 +217,19 @@ def on_create_alert_telegram_representative_async(self, alert_pk):
     )
     for message in messages_to_edit:
         edit_message.delay(message_pk=message.pk)
+
+
+@shared_dedicated_queue_retry_task(
+    autoretry_for=(Exception,), retry_backoff=True, max_retries=1 if settings.DEBUG else None
+)
+def on_alert_group_action_triggered_async(log_record_id):
```
Comment (matiasb) [inline engine/apps/telegram/tasks.py:225]: Just in case, don't forget to add the task to the celery routes setting.

**Review by matiasb — APPROVED** [review]
LGTM

File: engine/apps/telegram/tasks.py (hunk 2 above)
Comment (iskhakov) [reply engine/apps/telegram/tasks.py:223]: thank you, we a really cool test `common/tests/test_task_queue_assignment.py`

**Pull request merged**